xformers/components/attention/csrc/cuda/attention.cu

```diff
@@ -142,6 +146,36 @@ __device__ void compute_dot(
   }
 }
 
+/*
```

Comment:
nit: useful ?

Reply:
Not used, I was planning on doing some refactorings to make things simpler but didn't finish it and kept it there to eventually go back to it. I could just remove it though